manifests/charts/default/values.yaml

```diff
@@ -13,6 +13,9 @@ base:
   # Users should consider enabling this if they want full gateway-api validation but don't have other validation servers.
   validateGateway: false
 
+  # If disabled, permissions required to support custom k8s CAs will not be enabled.
+  enableCertificateSigningRequestControllers: true
```

Comment:
I think we already have existing APIs around this, do we need a new setting? I think EXTERNAL_CA enables it and PILOT_CERT_PROVIDER sets the signer name.

Also we have K8S_SIGNER, not sure why we have 2

cc @shankgan @irisdingbj

Reply:
yes, we used EXTERNAL_CA  and PILOT_CERT_PROVIDER for external CA integration.  Not sure K8S_SIGNER though. @shankgan might share more lights here?

Reply:
@irisdingbj @howardjohn : i removed the additional parameter, and reused EXTERNAL_CA and defaultCertProvider, could you please have a look now?